.github/workflows/maven.yml

```diff
@@ -0,0 +1,162 @@
+name: Java CI
+
+on: [push, pull_request]
```

Comment:
We need to run this on the `push` event in order to generate builds for testing outside of the context of a pull request, right?

Reply:
Yes, I believe that's the case.